storage/connect/reldef.cpp

```diff
@@ -533,7 +575,9 @@ PTABDEF OEMDEF::GetXdef(PGLOBAL g)
 #endif // 0
 
   // Is the library already loaded?
+#if !defined(_AIX)
   if (!Hdll && !(Hdll = dlopen(soname, RTLD_NOLOAD)))
+#endif
```

Comment:
aix `dlopen()` and shared library loading in general behaves very different from linux etc. so i added this aix specific dladdr function.

https://www.ibm.com/support/knowledgecenter/ssw_aix_53/com.ibm.aix.basetechref/doc/basetrf1/dlopen.htm